components/apimgt/org.wso2.carbon.apimgt.api/src/main/java/org/wso2/carbon/apimgt/api/APIConsumer.java

```diff
@@ -241,7 +241,7 @@ void cleanUpApplicationRegistration(String applicationName, String tokenType, St
      * @throws APIManagementException
      */
     Set<SubscribedAPI> getPaginatedSubscribedAPIs(Subscriber subscriber, String applicationName, int startSubIndex,
-                                                  int endSubIndex, String groupingId) throws APIManagementException;
+                                                  int endSubIndex, String groupingId, String organization) throws APIManagementException;
```

Comment:
We need this in  the javadoc comment. Please check all such cases.

Reply:
This is fixed now.